include/rocksdb/advanced_options.h

```diff
@@ -246,6 +246,11 @@ enum UpdateStatus {    // Return status For inplace update callback
   UPDATED         = 2, // No inplace update. Merged value set
 };
 
+enum PrepopulateBlobCache : uint8_t {
```

Comment:
Can this be an `enum class` instead of a plain `enum`?

Reply:
done